META.coq

```diff
@@ -6,6 +6,18 @@ version = "8.8"
 directory = ""
 requires = "camlp5"
 
+package "grammar" (
```

Comment:
Why's this moving?

Reply:
It is not just a move, see the diff, the old META entry for grammar was just wrong, `grammar` is just a plugin for `camlp5` [that we build outside the main coq tree] and thus doesn't belong in the main META dependency tree. In fact, in the Dune build we don't provide a grammar library and dynlink it, but we do distribute the whole driver [which is way faster for a start]

Reply:
In fact in my main working tree the parsing directory is gone altogether, hard to show the full thing tho, merging of this stuff is like, super slow, and if I stack people do complain.

Reply:
You could give a link to your main branch though so that people can have a look at the whole thing without feeling a pressing need to review.